code/game/objects/items/weapons/storage/toolbox.dm

```diff
@@ -13,6 +13,18 @@
 	origin_tech = "combat=1;engineering=1"
 	attack_verb = list("robusted")
 	hitsound = 'sound/weapons/smash.ogg'
+	var/image/hinges = "single_hinge"
+	var/old = 0
+
+/obj/item/weapon/storage/toolbox/New()
+	..()
+	if(!old)
+		if(prob(10))
+			hinges = "double_hinge"
+		else if(prob(1))
+			hinges = "triple_hinge"
+	add_overlay(image('icons/obj/storage.dmi', "[hinges]"))

```

Comment:
@coiax that'd actually be adding another overlay each update icons not a good idea 

also other stuff doesnt do that in update icons it's new aswell so ill not